docs/helm-chart-checks.md

```diff
@@ -36,14 +36,14 @@ The following table lists the default set of checks with details including the n
 | `has-kubeversion` | Mandatory | Checks that the `Chart.yaml` file of the Helm chart includes the `kubeVersion` field.
 | `contains-values-schema` | Mandatory | Checks that the Helm chart contains a JSON schema file (`values.schema.json`) to validate the `values.yaml` file in the chart.
 | `not-contains-crds` | Mandatory | Checks that the Helm chart does not include custom resource definitions (CRDs).
-| `not-contain-csi-objects` | Mandatory | Checks that the Helm chart does not include Container Storage Interface (CSI)[¹](https://gist.github.com/Srivaralakshmi/eb3c9bf1d65ec297035f4a8d26057620#-for-more-information-on-csi-see-container-storage-interface) objects.
+| `not-contain-csi-objects` | Mandatory | Checks that the Helm chart does not include Container Storage Interface (CSI) objects.
 | `images-are-certified` | Mandatory | Checks that the images referenced by the Helm chart are Red Hat-certified.
 | `helm-lint` | Mandatory | Checks that the chart is well formed by running the `helm lint` command.
 | `chart-testing` | Mandatory | Installs the chart and verifies it on a Red Hat OpenShift Container Platform cluster.
-| `contains-values` | Mandatory | Checks that the Helm chart contains the `values`[²](https://gist.github.com/Srivaralakshmi/eb3c9bf1d65ec297035f4a8d26057620#-for-more-information-on-the-values-file-see-values-and-best-practices-for-using-values) file.
+| `contains-values` | Mandatory | Checks that the Helm chart contains the `values`[¹](https://github.com/redhat-certification/chart-verifier/blob/main/docs/helm-chart-checks.md#for-more-information-on-the-values-file-see-values-and-best-practices-for-using-values) file.
```

Comment:
Can we have the `1.` before the text in the footnote to indicate that it is a footnote. The way it is rendered currently, it is not clear that the explanation is linked to [1]

Reply:
Done. Thanks for the suggestion.